Rakefile

```diff
@@ -122,229 +145,18 @@ end
 
 def each_gem
   each_child do |dir|
-    gem_dir = dir.split('/').last.to_sym
-    if REGEN_METADATA.has_key?(gem_dir)
+    #puts "Each child #{dir}"
+    gem_dir = dir.split('/').last
+    if get_config_file.has_key?(gem_dir)
+     # puts "Gem dir #{gem_dir}"
       yield gem_dir
     end
   end
 end
 
 REGEN_METADATA = {
-    autorest_loc: ENV.fetch('AUTOREST_LOC', '../../../autorest/binaries/net45/AutoRest.exe'),
+    autorest_loc: "node F:/Git/newautorest/autorest/src/autorest-core/dist/app.js", #autorest", #ENV.fetch('AUTOREST_LOC', '../../../autorest/binaries/net45/AutoRest.exe'),
     azure_sdk: {
```

Comment:
Should we move this to the config file as well and get rid off the REGEN_METADATA?